src/context/AppProvider/AppContext.tsx

```diff
@@ -136,8 +129,28 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
   }, [dispatch, prevWalletId, walletId])
 
   const marketDataPollingInterval = 60 * 15 * 1000 // refetch data every 15 minutes
+
+  // load top 1000 assets market data
+  // this is needed to sort assets by market cap
+  // and covers most assets users will have
+  const topAssetsMarketData = useFindAllMarketDataQuery(undefined, {
+    skip: !isConnected || portfolioLoadingStatus === 'loading' || modal || isLoadingLocalWallet,
+    pollingInterval: marketDataPollingInterval,
```

Comment:
We were missing polling for `findAll()` which yeah, now that we rely on it as a source of truth for top assets (as we should have), definitely makes sense to have